source/_integrations/ozw.markdown

```diff
@@ -65,3 +65,24 @@ this operation.
 | Service Data Attribute | Required | Description                                        |
 | ---------------------- | -------- | -------------------------------------------------- |
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
+Valid usercodes are at least 4 digits.
+
+| Service Data Attribute | Required | Description                             |
+| ---------------------- | -------- | --------------------------------------- |
+| `entity_id`            | yes      | Lock entity to set the usercode.        |
+| `code_slot`            | yes      | The code slot to set the usercode into. |
+| `usercode`             | yes      | The code to set in the slot.            |
+
+### Service `ozw.clear_usercode`
+
+This service will clear the usercode in code_slot X.
+Valid code_slots are 1-254.
```

Comment:
Missing word:

```
Valid `code_slots` are between 1-254.
```

Additionally, this comment can be part of the description of the service data attribute in the table above.

Currently, it is missing in the `ozw.set_usercode` description? Or doesn't it have that limitation?

Reply:
I'll get a PR to fix the services.yaml too, thanks for that.

Reply:
Here it is: https://github.com/home-assistant/home-assistant.io/pull/13960
Thank you for your review @frenck